site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -2,4 +2,79 @@
 api: runtime
 ---
 
-<!-- Intentionally blank -->
+## Examples
+
+### Use [`getURL`][getURL] to add an extension image to a page {: #example-getURL }
+
+Extensions can access their own resources using relative paths, but in order for the extension's
+assets to appear in a web page, you will need to specify the full URL of the asset (and expose it
+using the [web_accessible_resources][war] field).
+
+This example shows how a [content script][content] could add an image in the extension's package to
+the page that the content script has been [injected][content-inject] into.
+
+```js
+//// content.js
+
+{ // Block used to avoid setting global variables
+  let img = document.createElement('img');
+  img.src = chrome.runtime.getURL('logo.png');
+  document.body.append(img);
+}
+```
+
+### Use runtime messaging to communicate between background and content scripts {: #example-background-content-messaging }
+
+A basic example of content script injected into a page and background script passing messages to
```

Comment:
Consider:

A common pattern is for a background script to inject a content script into a page. In this case, the content and background scripts are running in separate contexts, and cannot invoke each other directly. Instead, the scripts can communicate by [passing messages](/docs/extensions/mv3/messaging/ but in footnote). The following example shows how...

Reply:
Ack. Rewriting the intro.